src/ostorlab/runtimes/local/models/models.py

```diff
@@ -572,6 +650,44 @@ class Asset(Base):
         sqlalchemy.Integer, sqlalchemy.ForeignKey("scan.id"), nullable=True
     )
 
+    @staticmethod
+    def create_from_assets_def(
+        scan: Scan, assets: Optional[List[base_asset.Asset]]
```

Comment:
1. I would use only scan_id  in the param.
2. can't we create an asset without scan ?

Reply:
Yes we can, using just create

Reply:
Applied